### DISCUSSION TITLE
【学习交流】请问为啥要做运营商的IP对比啊

### DISCUSSION THREAD

**Discussion by vastroc** [jeessy2/ddns-go#949] [Q&A]
检测到本地网卡IP发生变化，直接同步到运营商不就好了吗，还是说这样设计有更深层次的考虑

**Comment by jeessy2** [comment]
记录不存在新增，存在编辑，有些运营商新增/编辑调用的接口不同

**Reply by vastroc** [reply]
我的意思是，设计定时去请求运营商的接口的作用是什么呢，只需要定时检测本地IP变化就可以了哇，检测到变化就发API请求修改解析记录，这样就不用频繁请求运营商了

**Reply by jeessy2** [reply]
怕有其它程序或手动修改
